Doesn't support NextJS 9.0.5

Hey! I'm trying to apply your library for Next 9.0.5 and i get the error:
```
[ error ] ./node_modules/connected-next-router/es/patchRouter.js
Module not found: Can't resolve 'next/dist/next-server/lib/router/rewrite-url-for-export' in '{projectPath}/node_modules/connected-next-router/es'
```

That's because you're importing `rewriteUrlForNextExport` [over there](https://github.com/danielr18/connected-next-router/blob/master/src/patchRouter.ts#L7), but NextJS 9.0.5 doesn't exports it.

Updating to next@9.4.4 solves the problem.
Maybe you'll add some info about that or more readable error?

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -24,6 +24,7 @@ module.exports = {
     '@typescript-eslint/prefer-interface': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
     'no-empty-function': 'off',
+    'prefer-const': 'off',
     '@typescript-eslint/no-empty-function': 'off',
     '@typescript-eslint/explicit-function-return-type': [
       'error',

--- a/src/patchRouter.ts
+++ b/src/patchRouter.ts
@@ -1,10 +1,9 @@
 /* global __NEXT_DATA__ */
-
+declare const __NEXT_DATA__: any
 import { UrlObject } from 'url'
 import { RouterAction, BeforePopStateCallback } from './types'
 import { SingletonRouter, Router } from 'next/router'
 import { formatWithValidation } from 'next/dist/next-server/lib/utils'
-import { rewriteUrlForNextExport } from 'next/dist/next-server/lib/router/rewrite-url-for-export'
 
 type Url = UrlObject | string
 type HistoryMethod = 'replaceState' | 'pushState'
@@ -18,9 +17,10 @@ const patchRouter = (Router: RouterToPatch): (() => void) => {
     return Router.router.change(method, _url, _as, options).then((changeResult: boolean) => {
       if (changeResult) {
         if (process.env.__NEXT_EXPORT_TRAILING_SLASH) {
+          const rewriteUrlForNextExport = require('./utils/rewriteUrlForExport').rewriteUrlForNextExport
           // @ts-ignore this is temporarily global (attached to window)
           if (__NEXT_DATA__.nextExport) {
-            as = rewriteUrlForNextExport (as)
+            as = rewriteUrlForNextExport(as)
           }
         }
         Router?.router?.events.emit('connectedRouteChangeComplete', url, as, action)

--- a/src/utils/rewriteUrlForExport.ts
+++ b/src/utils/rewriteUrlForExport.ts
@@ -1,0 +1,13 @@
+export function rewriteUrlForNextExport(url: string): string {
+  const [pathname, hash] = url.split('#')
+  // tslint:disable-next-line
+  let [path, qs] = pathname.split('?')
+  if (path) {
+    path = path.replace(/\/$/, '')
+    // Append a trailing slash if this path does not have an extension
+    if (!/\.[^/]+\/?$/.test(path)) path += `/`
+  }
+  if (typeof qs === 'string') path += '?' + qs
+  if (typeof hash === 'string') path += '#' + hash
+  return path
+}

